public/markdown/Guide.md

```
@@ -1,35 +1,61 @@
+<details>
+<summary>How to switch Steve-xmh's AMLL TTML Tool into English</summary>
+
+![A GIF showing how to switch the TTML Editor webapp into English]({cdn_prefix}/useruploads/balint2201/guidemedia/LanguageSwitch.fbe2bf6f.webp)
+
+</details>
 
 ## 1. Upload a song
-![A GIF showing how to upload a song to the AMLL TTML Tool website.](https://wdeliverystatic.global.ic.spicylyrics.org/wclient/a-cdn/lyrprep/40f40d8e0eaa25df7eb0b29c60b83394ca6c567d.gif)
+![A GIF showing how to upload a song to the AMLL TTML Tool webapp.]({cdn_prefix}/useruploads/balint2201/guidemedia/AudioImport.9f8669ec.webp)
 > [!TIP]
 >  - You can download a song [here](https://spotidown.app/).
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)